src/svgs/icon-slide-left.svg

```diff
@@ -1,3 +1,3 @@
-<svg xmlns="http://www.w3.org/2000/svg" width="32" height="32" viewBox="0 0 32 32" fill="none">
-    <path fill-rule="evenodd" clip-rule="evenodd" d="M21.9427 24.3904C22.4634 24.9111 22.4634 25.7554 21.9427 26.2761C21.422 26.7968 20.5778 26.7968 20.0571 26.2761L10.7238 16.9427C10.2031 16.422 10.2031 15.5778 10.7238 15.0571L20.0571 5.72378C20.5778 5.20308 21.422 5.20308 21.9427 5.72378C22.4634 6.24448 22.4634 7.0887 21.9427 7.60939L13.5522 15.9999L21.9427 24.3904Z" fill="black"/>
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32" fill="none">
+    <path class="fill" fill-rule="evenodd" clip-rule="evenodd" d="M21.9427 24.3904C22.4634 24.9111 22.4634 25.7554 21.9427 26.2761C21.422 26.7968 20.5778 26.7968 20.0571 26.2761L10.7238 16.9427C10.2031 16.422 10.2031 15.5778 10.7238 15.0571L20.0571 5.72378C20.5778 5.20308 21.422 5.20308 21.9427 5.72378C22.4634 6.24448 22.4634 7.0887 21.9427 7.60939L13.5522 15.9999L21.9427 24.3904Z"/>
```

Comment:
Sorry, I forgot to explanate how the Icon components accept the colors. @liamcho

Reply:
no worries I noticed this bug after feature was released!